### PR TITLE
pridecat: add page

### PR DESCRIPTION
**Pull request by MeijeronFire**
- [x] The page (if new), does not already exist in the repository.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

**Version of the command being documented (if known):**

### CODE REVIEW

File: pages/linux/pridecat.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
+# pridecat
+
+> Like cat but more colorful :).
+> More information: <https://github.com/lunasorcery/pridecat>.
+
+- Output file in pride colors:
```
Comment (mfrw) [inline pages/linux/pridecat.md:6]: How about:
```suggestion
- Print the contents of a file in pride colors to the standard output:
```

File: pages/linux/pridecat.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+
+`pridecat {{file}}`
+
+- Output file in trans colors:
```
Comment (mfrw) [inline pages/linux/pridecat.md:10]:
```suggestion
- Print contents of a file in trans colors:
```

File: pages/linux/pridecat.md [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+
+`pridecat {{file}} --{{transgender|trans}}`
+
+- Alternate between lesbian and bisexual pride flags:
+
+`pridecat {{file}} --lesbian --bi`
+
+- Output file with the background colors changed:
```
Comment (mfrw) [inline pages/linux/pridecat.md:18]:
```suggestion
- Print contents of a file with the background colors changed:
```

**Review by mfrw — COMMENTED** [review]
I think we should be consisted with the original `cat` page :)

**Review by blueskyson — APPROVED** [review]

**Review by mfrw — APPROVED** [review]
Thank you @MeijeronFire for your contribution.
LGTM

**Pull request merged**